packit/specfile.py

```diff
@@ -34,6 +34,17 @@ def __init__(self, path: Union[str, Path], sources_dir: Union[str, Path] = ""):
         self._patch_id_digits: Optional[int] = None
         self._uses_autosetup: Optional[bool] = None
 
+    def has_autochangelog(self):
+        """
+        The %autochangelog macro is in the specfile?
+
+        :return bool
+        """
+        for value in self.spec_content.section("%changelog"):
+            if "%autochangelog" in value:
+                return True
```

Comment:
How about checking if there is any line that starts with `%autochangelog` instead? The current implementation would return `True` even for a comment line that contains `%autochangelog` (which would be incorrect).

Reply:
I'd rather choose the safe route:

* `%changelog` contains only `%autochangelog` entry → `True`
* `%changelog` contains `%autochangelog` and something else → raise an exception
* otherwise → `False`

Reply:
I would probably go with the safer way:

* `%changelog` contains only `%autochangelog` → `True`
* `%changelog` contains `%autochangelog` and something else → raise an exception
* otherwise → `False`

Reply:
Yes, you right.

Reply:
> I would probably go with the safer way:
> 
>     * `%changelog` contains only `%autochangelog` → `True`
> 
>     * `%changelog` contains `%autochangelog` and something else → raise an exception
> 
>     * otherwise → `False`

I am not raising an exception because I thought that maybe it is too much to raise it if, as an example, there is a comment inside the section.